src/database/migrations/1641994291086-add-posts-table.ts

```diff
@@ -6,15 +6,16 @@ export class AddPostsTable1641994291086 implements MigrationInterface {
   public async up(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(`
       CREATE TABLE posts (
-      id CHAR(36) PRIMARY KEY,
-      email VARCHAR(255) UNIQUE NOT NULL,
-      name VARCHAR(255) NOT NULL,
-      username VARCHAR(255) UNIQUE NOT NULL,
-      avatar VARCHAR(255),
-      password VARCHAR(255) NOT NULL,
-      address VARCHAR(255),
-      bio VARCHAR(255)
-    );`);
+        id CHAR(36) PRIMARY KEY,
```

Comment:
Don't use this way, try another way call migration API
Reference the bellow link
https://orkhan.gitbook.io/typeorm/docs/migrations#using-migration-api-to-write-migrations